### PR TITLE
mrc-1811 put unfinished features behind switches

### PR DESCRIPTION
**Pull request by hillalex**
This PR puts project copying and sharing behind switches.

Switches can be turned on programmtically by hard-coding to `true` [here](https://github.com/mrc-ide/hint/compare/mrc-1811?expand=1#diff-98e30f6f02ade1e1de8b350eac3cc51fR5) or by the user when the app is running by appending query strings to the url. This is for manual testing and demo-ing. 

Append the following query strings to see the hidden features:

* shareProject=true
* copyProject=true

### CODE REVIEW

File: src/app/static/src/tests/components/projects/projectHistory.test.ts [hunk 1]
```diff
@@ -77,9 +78,13 @@ describe("Project history component", () => {
         expectTranslated(v.at(4),"Load", "Charger", store);
         expect(v.at(4).find("a").attributes("href")).toBe("");
         expectTranslated(v.at(5), "Delete", "Supprimer", store);
-        expectTranslated(v.at(6),"Copy to a new project", "Copier dans un nouveau projet", store);
-        expect(v.at(6).find("a").attributes("href")).toBe("");
-
+        if (switches.copyProject) {
+            expectTranslated(v.at(6), "Copy to a new project", "Copier dans un nouveau projet", store);
```
Comment (Nicolas-Dolan) [inline src/app/static/src/tests/components/projects/projectHistory.test.ts:82]: I like that you can check both translations in one line :)

**Review by Nicolas-Dolan — COMMENTED** [review]
I'll copy this format when adding new features in future. I'll check back in once Travis fail has been fixed.

**Comment by hillalex** [comment]
Looks like a file has been deleted from ADR, causing integration tests to fail

File: src/app/static/src/tests/components/projects/projectHistory.test.ts (hunk 1 above)
Comment (hillalex) [reply src/app/static/src/tests/components/projects/projectHistory.test.ts:82]: this is our new test util method for testing translations - motivated by your more rigorous approach to testing translations than we had taken in the past!

File: src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ADRTests.kt [hunk 2]
```diff
@@ -109,39 +109,38 @@ class ADRTests : SecureIntegrationTests()
...
-    @EnumSource(IsAuthorized::class)
-    fun `can save shape from ADR`(isAuthorized: IsAuthorized)
-    {
-        val shape = extractUrl(isAuthorized, "inputs-unaids-geographic")
-        val result = testRestTemplate.postForEntity<String>("/adr/shape",
-                getPostEntityWithUrl(shape))
-        assertSecureWithSuccess(isAuthorized, result, "ValidateInputResponse")
-    }
-
+//
```
Comment (hillalex) [inline src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ADRTests.kt:112]: Looks like all adr files are being intermittently deleted (possibly while demo-ing the app during the unaids ref group meeting this afternoon) so have had to comment out all these tests for now.

File: src/app/static/src/tests/components/projects/projectHistory.test.ts [hunk 3]
```diff
@@ -77,9 +78,13 @@ describe("Project history component", () => {
         expectTranslated(v.at(4),"Load", "Charger", store);
         expect(v.at(4).find("a").attributes("href")).toBe("");
         expectTranslated(v.at(5), "Delete", "Supprimer", store);
-        expectTranslated(v.at(6),"Copy to a new project", "Copier dans un nouveau projet", store);
-        expect(v.at(6).find("a").attributes("href")).toBe("");
-
+        if (switches.copyProject) {
```
Comment (EmmaLRussell) [inline src/app/static/src/tests/components/projects/projectHistory.test.ts:81]: All the switching in the tests seems to be on copyProject, should there not also be a test that the shareProject component is/is not rendered?

**Review by EmmaLRussell — COMMENTED** [review]
Nice! Pity about the adr files, hopefully they'll be more stable tomorrow?

File: src/app/static/src/tests/components/projects/projectHistory.test.ts (hunk 3 above)
Comment (hillalex) [reply src/app/static/src/tests/components/projects/projectHistory.test.ts:81]: There apparently isn't a test for that at the moment! Can add one.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**